src/ReactiveUI.Fody.Analyzer.Test/ReactiveUI.Fody.Analyzer.Test.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net461</TargetFrameworks>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.9.4" />
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.2.1" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+        <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
+        <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
+    </ItemGroup>
+    <ItemGroup>
+      <ProjectReference Include="..\ReactiveUI.Fody.Analyzer\ReactiveUI.Fody.Analyzer.csproj" />
+      <ProjectReference Include="..\ReactiveUI.Fody.Helpers\ReactiveUI.Fody.Helpers.csproj" />
+      <ProjectReference Include="..\ReactiveUI.Fody\ReactiveUI.Fody.csproj" />
+      <ProjectReference Include="..\ReactiveUI\ReactiveUI.csproj" />
+    </ItemGroup>
+    <ItemGroup>
```

Comment:
This is likely not needed.

Reply:
It is actually needed for setting up the Roslyn compilation environment, which we need to compile our test code in.